doc/usage.adoc

```diff
@@ -7,65 +7,46 @@ terminal which has the following usage:
 
 [source,bash]
 ----
-usage: osivalidator [-h] [--data DATA] [--rules RULES] [--type {SensorView,GroundTruth,SensorData}] [--output OUTPUT] [--timesteps TIMESTEPS] [--debug] [--verbose] [--parallel] [--format {separated,None}]
+usage: osivalidator [-h] --data DATA [--rules RULES] [--type {SensorView,GroundTruth,SensorData}] [--output OUTPUT] [--timesteps TIMESTEPS] [--debug] [--verbose] [--parallel] [--format {None}]
                     [--blast BLAST] [--buffer BUFFER]
 
 Validate data defined at the input
 
+mandatory arguments:
+--data DATA           Path to the file with OSI-serialized data.
+
 optional arguments:
 -h, --help            show this help message and exit
---data DATA           Path to the file with OSI-serialized data.
 --rules RULES, -r RULES
-                        Directory with text files containig rules.
+                      Directory with text files containig rules.
 --type {SensorView,GroundTruth,SensorData}, -t {SensorView,GroundTruth,SensorData}
-                        Name of the type used to serialize data.
+                      Name of the type used to serialize data.
 --output OUTPUT, -o OUTPUT
-                        Output folder of the log files.
+                      Output folder of the log files.
 --timesteps TIMESTEPS
-                        Number of timesteps to analyze. If -1, all.
+                      Number of timesteps to analyze. If -1, all.
 --debug               Set the debug mode to ON.
 --verbose, -v         Set the verbose mode to ON.
---parallel, -p        Set parallel mode to ON.
---format {separated,None}, -f {separated,None}
-                        Set the format type of the trace.
+--parallel, -p        (Ignored) Set parallel mode to ON.
```

Comment:
While I totally agree to leave legacy options in the program, I would recommend to add a reason why they are ignored. This helps new users to ignore certain help texts more easily and not anticipate that this might be a future experimental setting: E.g. `(Legacy, ignored) Set parallel mode to ON.`

But of course, this is just something of taste and thus my personal opinion.

Reply:
If we keep the options as legacy, we should at least output a warning to the user.

Reply:
Right now I would just leave them as ignored, as they were options that had no real user-detectable effects (i.e. parallelization did not of course affect output, and not really even performance, same for the buffering). Given that we will likely remove most of them in the soonish next major release (but might resurrect parallel), I wouldn't put too much into the documentation of things. If this turns into a more permanent solution, more doc is warranted, but we are really hoping this is not the case.